include/picongpu/unitless/precision.unitless

```diff
@@ -66,6 +66,10 @@ namespace picongpu
     using float3_64 = precision64Bit::float3_X;
     using floatD_64 = precision64Bit::floatD_X;
 
+    // literals for short-hand notations
+    constexpr float_X operator""_X( long double x ) { return float_X( x ); }
```

Comment:
could you please write it on multiple lines?

Reply:
But... but... I like it like this :)